### PR TITLE
1651799 - added logrotate for output log file

### PR DESCRIPTION
**Pull request by lzap**
* [x] Nightly

Unsure about Debian tho, @mmoll ?

### CODE REVIEW

**Comment by ehelms** [comment]
Needs a rebase

**Comment by mmoll** [comment]
@lzap What kind of files would be there? I don't have any on my Ubuntu box with -tasks installed...
```
# ls -l /var/log/foreman/*.output
ls: cannot access '/var/log/foreman/*.output': No such file or directory
```

**Comment by lzap** [comment]
I will rebase once you guys come to agreement.

**Comment by ares** [comment]
@lzap please rebase, there's probably no objection, I suppose the question has been answered

**Comment by lzap** [comment]
Rebased!

File: packages/plugins/rubygem-foreman-tasks/foreman-tasks.logrotate [hunk 1]
```diff
@@ -0,0 +1,9 @@
+/var/log/foreman/*.output {
```
Comment (evgeni) [inline packages/plugins/rubygem-foreman-tasks/foreman-tasks.logrotate]: do we expect any other files to be called `*.output` besides `dynflow_executor.output`? If we don't, I'd prefer being explicit here and list `/var/log/foreman/dynflow_executor.output`.

**Comment by lzap** [comment]
Done thanks.

File: packages/plugins/rubygem-foreman-tasks/rubygem-foreman-tasks.spec [hunk 2]
```diff
@@ -128,6 +133,9 @@ exit 0
 %doc %{gem_instdir}/extra/dynflow-executor.example
 
 %changelog
+* Tue Feb 26 2019 Lukas Zapletal <lzap+rpm@redhat.com> 0.15-2
```
Comment (evgeni) [inline packages/plugins/rubygem-foreman-tasks/rubygem-foreman-tasks.spec]:
```suggestion
* Tue Feb 26 2019 Lukas Zapletal <lzap+rpm@redhat.com> 0.15.0-2
```

rpmlint is picky, and so am I ;-)

Comment (lzap) [reply packages/plugins/rubygem-foreman-tasks/rubygem-foreman-tasks.spec]: Oh thanks done.

**Comment by evgeni** [comment]
build: http://koji.katello.org/koji/taskinfo?taskID=175527

File: packages/plugins/rubygem-foreman-tasks/rubygem-foreman-tasks.spec [hunk 3]
```diff
@@ -117,6 +121,7 @@ exit 0
 %{foreman_apipie_cache_foreman}
 %{foreman_apipie_cache_plugin}
 %{foreman_dir}/script/foreman-debug.d/60-dynflow_debug
+%config(noreplace) %{_sysconfdir}/logrotate.d/%{gem_name}
```
Comment (lzap) [inline packages/plugins/rubygem-foreman-tasks/rubygem-foreman-tasks.spec]: Question, looks like the file ended up in:

```
# rpm -ql tfm-rubygem-foreman-tasks | grep logro
/opt/theforeman/tfm/root/etc/logrotate.d/foreman-tasks
```

How is this possible? I see not SCL prefix in the install and file section. Any hint @ekohl ?

Comment (evgeni) [reply packages/plugins/rubygem-foreman-tasks/rubygem-foreman-tasks.spec]: When the rpm is built inside the SCL the rpm macros point at paths inside the SCL too.

Comment (ekohl) [reply packages/plugins/rubygem-foreman-tasks/rubygem-foreman-tasks.spec]: You're looking for `_root_sysconfdir` so it's actually used.

**Pull request merged**